### PR TITLE
Create and Update JapanTimes.co.jp.xml

### PR DESCRIPTION
**Pull request by cschanaj**
This is a re-open of #8963 (closed) because of a mistake.

### CODE REVIEW

File: src/chrome/content/rules/JapanTimes.co.jp.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+		SSL peer certificate or SSH remote key was not OK:
+			 - www.japantimes.co.jp
+			 - realestate.japantimes.co.jp
+			 - support.japantimes.co.jp
+
+		Peer certificate cannot be authenticated with given CA certificates:
+			 - bookclub.japantimes.co.jp
+			 - genki.japantimes.co.jp
+			 - ij.japantimes.co.jp
```
Comment (J0WI) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: bookclub, genki and ij have an incomplete cert chain and redirect to HTTP

File: src/chrome/content/rules/JapanTimes.co.jp.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+			 - bugs.japantimes.co.jp
+			 - cms.japantimes.co.jp
+			 - features.japantimes.co.jp
+			 - ipm-archives.japantimes.co.jp
+			 - showcase.japantimes.co.jp
+			 - aomori.showcase.japantimes.co.jp
+			 - great-tamba.showcase.japantimes.co.jp
+			 - women.japantimes.co.jp
+
+		SSL peer certificate or SSH remote key was not OK:
```
Comment (J0WI) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: https-everywhere is not related to SSH

Comment (cschanaj) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: Sorry!! See https://github.com/EFForg/https-everywhere/pull/8973#discussion_r106060205.

File: src/chrome/content/rules/JapanTimes.co.jp.xml (hunk 1 above)
Comment (cschanaj) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: Fixed. 

For research purpose, why a `Incomplete certificate chain` has to be treated differently? When I use `curl` to test against the hosts, it throws an error before the redirections. However, Firefox does not. Is this a reason for that?

Comment (J0WI) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: Firefox caches the intermediate certificates, but if you access the site with a brand new profile you get a mismatch too.

Comment (cschanaj) [inline src/chrome/content/rules/JapanTimes.co.jp.xml]: I see. thanks!!

**Pull request merged**